lib/mrsk.rb

```diff
@@ -1,3 +1,8 @@
+require "zeitwerk"
+
+loader = Zeitwerk::Loader.for_gem
```

Comment:
From the [docs](https://github.com/fxn/zeitwerk#for_gem) adding `Zeitwerk::Loader.for_gem` should automatically introduce autoloading of the files under the `lib/` folder so I wonder if we still need to require them manually?

Also, I'm not 100% sure if we want to autoload all the files under `lib/` because from what I see [in the current entry point ](https://github.com/rails/mrsk/blob/35b42cc8859bd94eeadc985a68c96d3f2dc41090/lib/mrsk.rb#L4-L5) we don't require the `lib/cli` files so adding `Zeitwerk::Loader.for_gem` would change the existing behavior.

Reply:
Do you think we should just get Zeitwerk to autoload when running tests and leave it out of the lib code altogether?  As mentioned in the docs [here](https://github.com/fxn/zeitwerk#testing-compliance)